<launch>
    <arg name="sim_pepper_pose" default="0.0 0.0 0.0 0.0 0.0 0.0"/>
    <arg name="fruit_shape" default="0.08 0.035"/>
    <arg name="peduncle_shape" default="0.02 0.002"/>
    <arg name="xyz_noise" default="0.01"/>
    <arg name="pub_hz" default="10"/>
    <arg name="pub_topic" default="/fruit_coarse_pose"/>

    <node pkg="sim_fake_perception" type="fake_coarse_pe_pub.py" name="fake_coarse_pe_pub" output="screen">
        <param name="sim_pepper_pose" value="$(arg sim_pepper_pose)" />
        <param name="xyz_noise" value="$(arg xyz_noise)" />
        <param name="pub_hz" value="$(arg pub_hz)" />
        <param name="fruit_shape" value="$(arg fruit_shape)" />
        <param name="peduncle_shape" value="$(arg peduncle_shape)" />
        <param name="pub_topic" value="$(arg pub_topic)" />
    </node>
</launch>